### PR TITLE
Fix: HTTP Status Messages for lizmap servcie responses

### PR DESCRIPTION
**Pull request by rldhont**
Implement a static method Proxy::getHttpStatusMsg to get the HTTP Status Message from code to provide the right HTTP Status Messages in lizmap service responses.

* Funded by 3liz

### CODE REVIEW

**Comment by rldhont** [comment]
Cypress green

File: lizmap/modules/lizmap/lib/Request/Proxy.php [hunk 1]
```diff
@@ -136,6 +136,35 @@ public static function build($project, $params, $requestXml = null)
...
     }
 
+    /**
+     * Returns the HTTP Status Message.
+     *
+     * @param int $code the HTTP Status Code
+     *
+     * @return string $msg the HTTP Status Message
+     */
+    public static function getHttpStatusMsg($code)
```
Comment (mdouchin) [inline lizmap/modules/lizmap/lib/Request/Proxy.php:146]: Perhaps add more error code messages, such as 400, 405 and 500 like in https://github.com/3liz/lizmap-gobsapi-module/blob/master/gobsapi/controllers/apiController.php#L10 ?

**Review by mdouchin — CHANGES_REQUESTED** [review]

File: lizmap/modules/lizmap/lib/Request/Proxy.php (hunk 1 above)
Comment (rldhont) [reply lizmap/modules/lizmap/lib/Request/Proxy.php:146]: @mdouchin fixed

**Comment by rldhont** [comment]
Rebase to get cypress updated to 9.5.0 and Cypress tests are still green.

**Review by laurentj — CHANGES_REQUESTED** [review]

File: lizmap/modules/lizmap/lib/Request/Proxy.php [hunk 2]
```diff
@@ -136,6 +136,50 @@ public static function build($project, $params, $requestXml = null)
...
+    {
+        switch ($code) {
```
[the remark this answers is not in the document]
Comment (rldhont) [reply lizmap/modules/lizmap/lib/Request/Proxy.php]: Changes requested done.

**Comment by rldhont** [comment]
@laurentj changes done and cypress tests are still green

**Review by laurentj — APPROVED** [review]

**Review by mdouchin — APPROVED** [review]

**Pull request merged**